Octokit/Http/SimpleJsonSerializer.cs

```diff
@@ -137,6 +155,30 @@ static string RemoveHyphenAndUnderscore(string stringValue)
                         p => p.JsonFieldName,
                         p => new KeyValuePair<Type, ReflectionUtils.SetDelegate>(p.Type, p.SetDelegate));
             }
+
+            private static Type GetPayloadType(string activityType)
```

Comment:
Would it be worth extracting this to an extension method for better readability?